packages/custom_lint/test/create_project.dart

```diff
@@ -170,7 +170,7 @@ Directory createLintUsage({
       .map(
         (e) => '''
   ${e.key}:
-    path: ${e.value}
+    path: ${e.value.toFilePath()}
```

Comment:
Path in `pubspec.yaml` needs to be a file path rather than a URI. This fails when running the test project in a separate process.